app/src/vuex/modules/connection.js

```diff
@@ -150,8 +154,29 @@ export default function ({ apollo }) {
       }
       commit("setAddressType", network.address_creator)
       dispatch(`checkForPersistedSession`) // check for persisted session on that network
+      // if the network is a pokadot network
+      if (network.network_type === `polkadot`) {
+        actions.connectToPolkadotAPI({ commit }, { networkId: network.id })
```

Comment:
```suggestion
        actions.connectToPolkadotAPI({ commit }, network
```

this way u save urself the extra network lookup

Reply:
true!